CONTRIBUTING.md

```diff
@@ -231,6 +231,19 @@ From the RMarkdown Cookbook [bibliographies chapter](https://bookdown.org/yihui/
 You can obtain formatted `LaTeX` reference entry following the instructions below and copying and pasting the whole thing in `reference.bib`.
 The references in `reference.bib` should be kept in alphabetical order (this will reduce the chances of adding duplicates).
 
+Automatic reference renders (like those recommended below) don't maintain consistent syntax (though they usually will still work).
+You may/will still need to do some manual edits to maintain consistency with the current `references.bib`-listed sources.
+
+**Adding sources rules:**  
+
+- All sources need to have URLs.
+- Where applicable, sources should have doi's.
+- `{}` are preferred to `" "`.
+- All lower case tags are preferred to upper case (eg. `title` NOT `Title`).
+- Spaces before and after `=` are preferred.
```

Comment:
I don't really think we need these rules, as they seem overly prescriptive to me and don't affect rendering.

More helpful to suggest may be making sure the capitalization of tool and gene names will be preserved by judicious use of `{}`. 

I noted using this that `texmed` does bad stuff with titles. I just looked for alternatives, and https://www.doi2bib.org/ seems much better (and more general)